package.json

```diff
@@ -26,13 +26,13 @@
   },
   "homepage": "https://github.com/jfgreffier/playwright-relative-selector#readme",
   "devDependencies": {
-    "eslint": "^7.10.0",
-    "eslint-config-prettier": "^6.12.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "express": "^4.17.1",
-    "jest": "^26.4.2",
-    "playwright": "^1.4.2",
-    "prettier": "^2.1.2",
-    "promise": "^8.1.0"
+    "eslint": "~7.14.0",
+    "eslint-config-prettier": "~6.15.0",
+    "eslint-plugin-prettier": "~3.1.4",
+    "express": "~4.17.1",
+    "jest": "~26.6.3",
+    "playwright": "~1.6.2",
```

Comment:
Can you set it to `~1.4.2`, do a npm install and commit the package-lock.json ? In this way, we can verify that dependabot performs the update on next run